src/test/scala/ai/privado/languageEngine/kotlin/tagger/collection/KotlinCollectionTaggerTest.scala

```diff
@@ -0,0 +1,59 @@
+package ai.privado.languageEngine.kotlin.tagger.collection
+
+import ai.privado.cache.RuleCache
+import ai.privado.model.*
+import io.shiftleft.semanticcpg.language.*
+
+class KotlinCollectionTaggerTest extends KotlinTaggingTestBase {
+  private val collectionRule = List(
+    RuleInfo(
+      "Collections.Spark.HttpFramework",
+      "Spark Java Http Framework Endpoints",
+      "",
+      FilterProperty.METHOD_FULL_NAME,
+      Array(),
+      List(".*.get.*"),
+      false,
+      "",
+      Map(),
+      NodeType.REGULAR,
+      "",
+      CatLevelOne.COLLECTIONS,
+      catLevelTwo = Constants.default,
+      Language.KOTLIN,
+      Array()
+    )
+  )
+
+  "Simple Spark Http sample" should {
+    val ruleCache: RuleCache = getRuleCache
+    val fileContents = """
+        |import spark.*;
+        |
+        |fun main(args: Array<String>) {
+        |    Spark.get("/hello") { req, res ->
+        |        "Hello Spark Kotlin!"
+        |    }
+        |}
+        |""".stripMargin
+    initCpg(fileContents)
+    "should tag get collection endpoint" in {
+      cpg.call.methodFullName(".*.get.*").l.size shouldBe 1
+      cpg.call.head.code.contains("Spark") shouldBe true
+
+      val collectionTagger = new KotlinCollectionTagger(cpg, ruleCache)
+      collectionTagger.createAndApply()
+
+      val ingressRules = collectionTagger.getIngressUrls()
+      ingressRules should contain("\"/hello\"")
```

Comment:
This test case is incomplete, 
- Need to check if the call node `Spark.get` is tagged as CollectionPoint with correct endpoint
- Example, we can check on if the id is tagged as `Collections.Spark.HttpFramework` and we have a MethodEndpoint calculated for this node similar to how we have checked for `ingressRules`

Reply:
Added these to the test.